app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<h2 id='login-heading'>Log in</h2>
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %><br />
+    <%= f.password_field :password, autocomplete: "current-password" %>
+  </div>
+
+  <% if devise_mapping.rememberable? %>
+    <div class="field">
+      <%= f.check_box :remember_me %>
+      <%= f.label :remember_me %>
+    </div>
+  <% end %>
+
+  <div class="actions">
+    <%= f.submit "Log in" %>
+  </div>
+<% end %>
+
+<%= render "devise/shared/links" %>
```

Comment:
- [ ] Kindly ensure the `Signup` and `Login` buttons span the same length as the input fields so as to match the given design.

Given wireframe
![image](https://github.com/tan12082001/project-blog-app/assets/81932478/a28ce175-d817-4483-92dd-a742b51cbb2c)

Your design
![image](https://github.com/tan12082001/project-blog-app/assets/81932478/bc51a6d1-8d9f-47a6-bfb5-ffb8a404ef65)

Login page
![image](https://github.com/tan12082001/project-blog-app/assets/81932478/eee0d00c-ea2f-4676-9612-3369a2423f13)